app/javascript/components/zone-form/zone-form.schema.js

```diff
@@ -0,0 +1,120 @@
+import { componentTypes } from '@@ddf';
+
+const createSchema = () => ({
+    fields: [
+        {
+            component: componentTypes.SUB_FORM,
+            name: 'zone-information-subform',
+            title: __('Zone Information'),
+            fields: [
+                {
+                    component: componentTypes.TEXT_FIELD,
+                    id: 'name',
+                    name: 'name',
+                    label: __('Name'),
+                    maxLength: 50,
+                },
+                {
+                    component: componentTypes.TEXT_FIELD,
+                    id: 'description',
+                    name: 'description',
+                    label: __('Description'),
+                    maxLength: 50,
+                },
+                {
+                    component: componentTypes.TEXT_FIELD,
+                    id: 'proxy_server_ip',
+                    name: 'proxy_server_ip',
+                    label: __('SmartProxy Server IP'),
+                    maxLength: 50,
+                }         
+            ]
+        },
+        {
+            component: componentTypes.SUB_FORM,
+            name: 'ntp-servers-subform',
+            title: __('NTP Servers'),
+            fields: [
+                { 
+                    component: componentTypes.TEXT_FIELD,
+                    id: 'ntp_server_1',
+                    name: 'ntp_server_1',
+                    label: __('Servers'),
```

Comment:
I don't think you need this label here, the `SUB_FORM` above has a label and it is redundant. Either remove it, or set a label on each with an index ... your call. My concern here is a11y, that one field would have a label an others won't. IMO they should be consistent to not confuse the screen reader.

Reply:
I pushed new changes fixing the issues mentioned above. I removed the redundant Server label.